lib/glific_web/resolvers/tickets.ex

```diff
@@ -71,4 +71,15 @@ defmodule GlificWeb.Resolvers.Tickets do
   def fetch_support_tickets(_, args, _) do
     {:ok, Tickets.fetch_support_tickets(args)}
   end
+
+  @doc """
+  update multiple tickets
+  """
+  @spec update_bulk_ticket(Absinthe.Resolution.t(), %{input: map()}, %{context: map()}) ::
+          {:ok, any} | {:error, any}
+  def update_bulk_ticket(_, %{input: params}, _) do
+    with {:ok, ticket} <- Tickets.update_bulk_ticket(params) do
+      {:ok, %{ticket: ticket}}
```

Comment:
instead of {:ok, ticket: ticket} we can return success message as we are updating multiple tickets and returning only single does not make sense